UnityProject/Assets/Scripts/Sprite Handler/SpriteHandler.cs

```diff
@@ -58,139 +71,91 @@ public void PushTexture()
 				TryInit();
 			}
 
-			if (spriteIndex < Infos.List.Count)
-			{
-				if (VariantIndex < Infos.List[spriteIndex].Count)
-				{
-					if (animationIndex < Infos.List[spriteIndex][VariantIndex].Count)
-					{
-						SetSprite(Infos.List[spriteIndex][VariantIndex][animationIndex]);
-						TryToggleAnimationState(Infos.List[spriteIndex][VariantIndex].Count > 1);
-					}
-					else
-					{
-						spriteRenderer.sprite = null;
-						TryToggleAnimationState(false);
-					}
-				}
-				else
-				{
-					spriteRenderer.sprite = null;
-					TryToggleAnimationState(false);
-				}
-			}
-			else
+			if (spriteIndex < Infos.List[spriteIndex].Count &&
```

Comment:
I get an ArgumentOutOfRangeException here, due to Infos.List[spriteIndex]. Reproducible by going into botany as host player, opening the crate, and picking up  some seed packets.

It's because that first if was removed, if (spriteIndex < Infos.List.Count) still needs to be checked

Reply:
Oops, typo on my part.  Sorry about that.